IssueTracker/IssueTrackerAPI/Controllers/FileController.cs

```diff
@@ -38,22 +40,26 @@ public IActionResult GetFileBlob(string name)
     [HttpPost]
     public async Task<IResult> PostFile([FromForm] IFormFile formFile, [FromForm] int? issueId, [FromForm] int? commentId)
     {
+        if (issueId == null && commentId == null) return Results.BadRequest("can't both be null");
+        if (issueId <= 0 || commentId <= 0) return Results.BadRequest("ids are greater than 0");
         var fileId = Guid.NewGuid();
         var bolbFileName = $"{fileId}{Path.GetExtension(formFile.FileName)}";
         var file = formFile.OpenReadStream();
-        //_bolbData.Upload(file, bolbFileName);
+        _bolbData.Upload(file, bolbFileName);
         var fileName = formFile.FileName;
         var group = "Mihai";
-        var fileSize = formFile.Length / 1000;
+        var fileSize = formFile.Length / 1024;
         var fileType = formFile.ContentType;
         var metaDataRequest = new MetaDataRequest(fileId.ToString(), group, fileName, fileType, fileSize);
         var metaDataReq = _mapper.Map<MetaDataReq>(metaDataRequest);
         try
         {
             await _repository.CreateAsync(metaDataReq);
-            FileModel fileModel = new FileModel(fileId.ToString());
-            fileModel.CommentId = commentId;
-            fileModel.IssueId = issueId;
+            FileModel fileModel = new FileModel();
```

Comment:
is the model suffix needed for the 'FileModel' type?

Reply:
changed to File